lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -843,7 +843,6 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
         verify rGetTx
             [ expectResponseCode HTTP.status200
             , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
-            , expectField (#status . #getApiT) (`shouldBe` Pending)
```

Comment:
👍 But we haven't seen this being flaky yet?

Reply:
I had a failure locally... so it'd be just a matter of time I believe.